integtest/src/main/java/org/corfudb/universe/cluster/vm/VmCluster.java

```diff
@@ -0,0 +1,33 @@
+package org.corfudb.universe.cluster.vm;
+
+import com.google.common.collect.ImmutableList;
+import org.corfudb.universe.cluster.Cluster;
+import org.corfudb.universe.cluster.ClusterException;
+import org.corfudb.universe.service.Service;
+
+/**
+ * This implementation provides a Cluster servers each of which a VM machine.
+ * Deploy is going to deploy a list of servers
+ */
+public class VmCluster implements Cluster {
+
+    @Override
+    public Cluster deploy() throws ClusterException {
+        throw new UnsupportedOperationException("Not implemented");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Not implemented" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)